### PR TITLE
docs: configure_sync_batchnorm, amp, readme python/conda badge

### PR DESCRIPTION
**Pull request by ydcjeff**
## What does this PR do?

Fixes https://pytorch-lightning.readthedocs.io/en/latest/lightning-module.html#configure-sync-batchnorm
Previews from PR
- https://55409-178626720-gh.circle-artifacts.com/0/html/lightning-module.html#use-amp
- https://55409-178626720-gh.circle-artifacts.com/0/html/lightning-module.html#configure-sync-batchnorm
- https://55409-178626720-gh.circle-artifacts.com/0/html/lightning-module.html#on-test-batch-start

Changes conda downloads badges to 

Conda downloads

Add

PyPI - Python Version

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by ydcjeff** [comment]
Found out `use_amp` so added it and `on_test_*` were changed to align with the below `on_train_*` and `on_val_*`

**Review by rohitgr7 — APPROVED** [review]

File: docs/source/lightning-module.rst [hunk 1]
```diff
@@ -890,6 +890,10 @@ Pointer to the trainer
 
 ------------
 
+use_amp
+~~~~~~~
+True if using amp
+
 use_ddp
```
Comment (rohitgr7) [inline docs/source/lightning-module.rst:897]:
```suggestion
use_amp
~~~~~~~
True if using amp

------------

use_ddp
```

**Review by SkafteNicki — APPROVED** [review]

**Comment by Borda** [comment]
@ydcjeff I would pypi keep total downloads, so it is consistent with conda...

**Comment by ydcjeff** [comment]
> @ydcjeff I would pypi keep total downloads, so it is consistent with conda...

Yea, I also would like to, but I couldn't change to `pypi downloads` on pepy.tech
I searched thru the issue on https://github.com/psincraian/pepy, currently it doesn't have option to change

**Review by justusschock — APPROVED** [review]

**Comment by ydcjeff** [comment]
@Borda can we merge this?

File: docs/source/lightning-module.rst [hunk 2]
```diff
@@ -896,6 +896,12 @@ Pointer to the trainer
 
 ------------
 
+use_amp
+~~~~~~~
+True if using amp
```
Comment (Borda) [inline docs/source/lightning-module.rst:901]:
```suggestion
True if using Automatic Mixed Precision (AMP)
```

**Review by Borda — APPROVED** [review]
